TF_Settings_and_Tooling_Unity/Assets/TouchFree/SettingsUI/Scenes/ServiceUI.unity

```diff
@@ -2492,6 +2492,11 @@ PrefabInstance:
       propertyPath: m_LocalEulerAnglesHint.z
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 7125972924714312439, guid: dc0e884672fe42e49b02672301208def,
+        type: 3}
+      propertyPath: handsCameraObject
```

Comment:
It looks like the `handCameraObject` property has been has lost it's link in the scene by mistake. This is preventing the hands render from displaying

Reply:
This was already fixed, but I accidentally tested against an earlier commit